src/geo/leaflet/leaflet-torque-layer-view.js

```diff
@@ -5,7 +5,7 @@ var LeafletLayerView = require('./leaflet-layer-view');
 var TorqueLayerViewBase = require('../torque-layer-view-base');
 var util = require('cdb.core.util');
 
-var LeafletTorqueLayer = function (layerModel, leafletMap, mapModel) {
+var LeafletTorqueLayer = function (layerModel, leafletMap, mapModel, showLimitErrors) {
```

Comment:
do we really need this argument here? in the end we're making use of `this.showLimitErrors`

Reply:
Yes, because in the next line, we have to pass `showLimitErrors` to `LeafletLayerView`

```
LeafletLayerView.apply(this, arguments);
```